src/Admin.js

```diff
@@ -10,11 +10,12 @@ import crudSaga from './sideEffect/saga';
 import CrudRoute from './CrudRoute';
 import Layout from './mui/layout/Layout';
 
-const Admin = ({ restClient, appLayout = Layout, dashboard, children }) => {
+const Admin = ({ restClient, appLayout = Layout, dashboard, children, title = 'Admin on REST' }) => {
     const resources = React.Children.map(children, ({ props }) => props);
     const firstResource = resources[0].name;
     const sagaMiddleware = createSagaMiddleware();
     const reducer = combineReducers({
+        ui: () => ({ title }), // NOTE: maybe this could be a good fit for redux-ui ?
```

Comment:
I don't think we should use state for that. It's not going to change.

I understand that it's hard to pass to the Layout component, but I prefer that we use a HOC just for that.